src/main/java/org/spongepowered/api/command/parameter/managed/clientcompletion/ClientCompletionTypes.java

```diff
@@ -51,9 +51,13 @@
     public static final DefaultedRegistryReference<ClientCompletionType> DECIMAL_NUMBER = ClientCompletionTypes.key(ResourceKey.sponge("decimal_number"));
 
     /**
-     * Indicates to the client that the {@link ValueParser} will parse Json.
+     * Indicates to the client that the {@link ValueParser} will parse SNBT
+     * (the string format for NBT).
+     *
+     * <p>This format somewhat resembles <em>JSON</em>, but supports unquoted
+     * keys, and can unambiguously distinguish every NBT tag type.</p>
      */
-    public static final DefaultedRegistryReference<ClientCompletionType> JSON = ClientCompletionTypes.key(ResourceKey.sponge("json"));
+    public static final DefaultedRegistryReference<ClientCompletionType> SNBT = ClientCompletionTypes.key(ResourceKey.sponge("snbt"));
```

Comment:
Unsure about how to refer to this. While SNBT is a name for the format, and used on the MC wiki, the closest API type would be `DataContainer` -- would it make sense to call it `DATA_CONTAINER` instead?

Reply:
I wouldn’t say datacintainer is quite up to snuff, it does not currently restrict that offering anything is within the limits of being compatible with a tag compound (some parts are translated specially). But it’s been on my todo list to revamp the api to be closer to that of having DataView like a ConfigNode while it’s implementation being closer to tags.

Reply:
Ideally, it shouldn't refer to NBT. I'd probably just call it "DATA" because that's what it is, it _is_ a data blob.

Reply:
I think portraying the format as anything as vague as `DATA` provides a false sense of abstraction. The client does parse this as a SNBT compound for the purposes of completion, so it cannot be any arbitrary string, and knowning the format used is important for plugins understanding the reasons for using it. I think this could *become* something like `DATA_CONTAINER` if gabe has the time to make the changes he wants to though. For now I'll keep it as `SNBT` since that is, while not ideal, not wrong, but I'm not opposed to changing it in the future.